examples/how_to/get_started.py

```diff
@@ -196,50 +199,91 @@
 print("Units found by tridesclous:", sorting_TDC.get_unit_ids())
 print("Units found by spyking-circus2:", sorting_SC2.get_unit_ids())
 
-# If a sorter is not installed locally, we can also avoid installing it and run it anyways, using a container (Docker or Singularity). For example, let's run `Kilosort2` using Docker:
+# If a sorter is not installed locally, we can also avoid installing it and run it anyways, using a container (Docker or Singularity).
+# To do this, you will need to install Docker. More information [here](https://spikeinterface.readthedocs.io/en/latest/modules/sorters.html?highlight=docker#running-sorters-in-docker-singularity-containers).
+# Let's run `Kilosort2` using Docker:
 
 sorting_KS2 = ss.run_sorter(sorter_name="kilosort2", recording=recording_preprocessed, docker_image=True, verbose=True)
 print(sorting_KS2)
 
-# SpikeInterface provides a efficient way to extract waveforms from paired recording/sorting objects.
-# The `extract_waveforms` function samples some spikes (by default `max_spikes_per_unit=500`)
-# for each unit, extracts their waveforms, and stores them to disk. These waveforms are helpful to compute the average waveform, or "template", for each unit and then to compute, for example, quality metrics.
+# For postprocessing SpikeInterface pairs recording and sorting objects into a `SortingAnalyzer` object.
+# The `SortingAnalyzer` can be loaded in memory or saved in a folder. Here, we save it in binary format.
+
+sa_TDC = si.create_sorting_analyzer(sorting_TDC, recording_preprocessed, format='binary_folder', folder='sa_TDC_binary')
+
+# This folder is where all the postprocessing data will be saved such as waveforms and templates. Let's calculate
+# some waveforms. When doing this, the function samples some spikes (by default `max_spikes_per_unit=500`)
+# for each unit, extracts their waveforms, and stores them to disk in `extensions/waveforms`.
+# These waveforms are helpful to compute the average waveform, or "template", for each unit and then to compute, for example, quality metrics.
+# Computations with the `SortingAnalyzer` object are done using the `compute` method:
 
 # +
-we_TDC = si.extract_waveforms(recording_preprocessed, sorting_TDC, "waveforms_folder", overwrite=True)
-print(we_TDC)
+sa_TDC.compute("random_spikes")
```

Comment:
Maybe we need at least one example showing how you can tune parameters (max_spikes_per_unit=300) or something like that, but not sure since this is suppose to be our simplest tutorial.... Would be curious to see others opinions....

Reply:
Yes, I don't think it's confusing to add some parameters. I've got for computing a few things with default options, then a few things with some random parameters.

Reply:
I would rather build some tunning tutorials in the "how to section" and then link them here once we have them.